backend/src/handlers/transfer_operations.py

```
@@ -35,325 +41,339 @@ def get_transaction_by_id(transaction_id: uuid.UUID) -> Optional[Transaction]:
+            if success:
+                successful_pairs.append({
+                    "outgoingTransactionId": outgoing_tx_id,
+                    "incomingTransactionId": incoming_tx_id
+                })
+            else:
+                failed_pairs.append({
+                    "pair": pair,
+                    "error": "Failed to mark as transfer pair"
+                })
+                
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)